nilearn/datasets/utils.py

```diff
@@ -950,3 +950,15 @@ def make_fresh_openneuro_dataset_urls_index(
               "to update the file without breaking the fetcher download link."
               .format(urls_path))
     return urls_path, urls
+
+
+def load_sample_motor_activation_image():
+    """Load a single functional image showing motor activations.
+
+    Returns
+    -------
+    str
+        Path to the sample functional image.
+    """
+
+    return os.path.join(_PACKAGE_DIRECTORY, "data", "image_10426.nii.gz")
```

Comment:
Not super important, but another possibility would be to rely on pathlib for this and eventually cast to string when returning:

```python
from pathlib import Path

return str(Path(__file__).parent / "data" / "image_10426.nii.gz")
```

Reply:
Updated to what you suggested, thanks!